Mathlib/Analysis/Analytic/Polynomial.lean

```diff
@@ -74,4 +74,10 @@ theorem AnalyticOn.eval_linearMap' (f : σ → E →ₗ[𝕜] B) (p : MvPolynomi
 theorem AnalyticOn.eval_mvPolynomial [Fintype σ] (p : MvPolynomial σ 𝕜) :
     AnalyticOn 𝕜 (eval · p) Set.univ := AnalyticOn.eval_linearMap (.id (R := 𝕜) (M := σ → 𝕜)) p
 
+theorem MvPolynomial.continuous_eval
+    {X : Type*} [TopologicalSpace X] [CommSemiring X] [TopologicalSemiring X]
+    (p : MvPolynomial σ X) :
+    Continuous fun x ↦ (eval x) p := by
+  continuity
```

Comment:
#find_home suggests [Mathlib.Topology.Algebra.Module.Determinant] and I'm not sure whether there's better place ...

Reply:
Yeah, I'm not sure that's better.

Reply:
A better home would be a new file `Topology/Algebra/MvPolynomial` with lemmas about `eval`, `eval₂` etc, see `Topology/Algebra/Polynomial`.